docs/metrics/confusion-matrix.md

```diff
@@ -35,7 +31,26 @@ imbalances.
 
 The implementation of a confusion matrix depends on whether the workflow concerns one or more classes.
 
-### Single-Class
+!!!warning
+    The Confusion Matrix feature in Kolena supports visualizing comparisons
+    between a single predicted value and a single ground truth value. If you're
+    having trouble generating or customizing plots for these visualizations, feel free
+    to reach out to your Implementation Engineer for guidance. Alternatively, you
+    can contact us at [contact@kolena.com](mailto:contact@kolena.com)! for further support.
```

Comment:
Extra "!" after the email address

nit - I feel like this can be made more concise like "feel free to reach out to the Kolena team for support" as our typical customer would already have points of contact.

Reply:
Simplified this to align.